js/code1.js

```diff
@@ -0,0 +1,18 @@
+function code(){
+    let n = prompt("enter a number n :");
+    arr = [];
+    for (let i = 0; i < n; i++) {
+        arr.push(parseInt(Math.random()*20));

```

Comment:
This is off by 1. generates number between 0 and 19 instead of generating between 1 and 20.